pkgs/development/tools/backblaze-b2/default.nix

```diff
@@ -14,6 +14,10 @@ python3Packages.buildPythonApplication rec {
     hash = "sha256-10c2zddALy7+CGxhjUC6tMLQcZ3WmLeRY1bNKWunAys=";
   };
 
+  patches = [
+    ./help.patch
```

Comment:
It would be nice to link to the upstream PR. Or even using `fetchPatch` with the upstream commit as a source. But I'm not too concerned with carrying this tiny patch.